README.md

```diff
@@ -11,31 +11,36 @@ This repository contains the code that composes the io-web-profile front-end.
 
 This is Next.js 13 with app route project
 ## Prerequisites
+### Env
 In order to run the ``io-web-profile`` front-end locally you need the following tool installed on your machine.
 
 - ``Node.js 18.16.1``
 - ``yarn 1.22``
 
-In order to authenticate the login you need the hub-spid-login service provider. 
+The preferred way to set up the local environment is using nodenv to manage Node.js installation and corepack (included with Node.js) to manage the installation of yarn.
 
-https://github.com/pagopa/hub-spid-login-ms/tree/master
+### Login SP
 
-Follow the documentation of hub-spid-login to properly run the service.
+Given that the login for io-web-profile relies on the JWT generated by hub-spid-login, it is necessary to run hub-spid-login-ms service in the background.
 
-To establish the authentication and error binding between SP and IO Web Profile, you need to modify the default values inside the hub-spid-login .env file as follows:
+You can find the hub-spid-login project at this GitHub repository: https://github.com/pagopa/hub-spid-login-ms
 
-ENDPOINT_ERROR=http://localhost:3000/accedi/errore
-ENDPOINT_SUCCESS=http://localhost:3000/
+To ensure proper execution of the service, please follow the documentation provided by hub-spid-login. Additionally, make the following modifications to the default values within the hub-spid-login .env file:
 
-Note: The port used here is the same one used to expose the web app.
+``ENDPOINT_ERROR=http://localhost:3000/accedi/errore``
+``ENDPOINT_SUCCESS=http://localhost:3000/it/accedi
+``
+``
+ENABLE_JWT=true``
```

Comment:
you can add a major level of description, adding some like this: in order to enable JWTs we recommend following this part of the documentation https://github.com/pagopa/hub-spid-login-ms#jwt-support

Reply:
Descrption changed